manifests/a/Ablaze/Floorp/10.14.0/Ablaze.Floorp.installer.yaml

```diff
@@ -0,0 +1,32 @@
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.version.1.4.0.schema.json
+PackageIdentifier: Ablaze.Floorp
+PackageVersion: 10.14.0
+Platform:
+- Windows.Desktop
+MinimumOSVersion: 10.0.17763.0
+Commands:
+- floorp
+Protocols:
+- http
+- https
+FileExtensions:
+- htm
+- html
+- url
+ReleaseDate: 2023-06-04
+Installers:
+- Architecture: x64
+  Scope: machine
+  InstallerLocale: en-US
+  InstallerType: nullsoft
+  InstallModes:
+  - silent
+  - silentWithProgress
+  UpgradeBehavior: install
+  InstallerUrl: https://github.com/Floorp-Projects/Floorp/releases/download/v10.14.0/floorp-win64.installer.exe
+  InstallerSha256: 6328904ACC0A4E8EA8246E1B258DF529EA31F8071EC0197F3E20F378F4981B8B
+  AppsAndFeaturesEntries:
+  - DisplayName: Ablaze Floorp (x64 en-US)
+    ProductCode: Ablaze Floorp 10.14.0 (x64 en-US)

```

Comment:
```suggestion
```
AppsAndFeaturesEntries are not necessarily needed for this package since they would require manual updating for every new version. The PackageName and Publisher already provide a good match to WinGet for upgrade matching